.github/workflows/docker-build-push-gcp.yml

```
@@ -26,10 +26,14 @@ on:
         required: false
         type: string
         default: "."
+      file:
+        required: false
+        type: string
+        default: "{context}/Dockerfile"
```

```suggestion
        default: "./Dockerfile"
```
Not sure if we can reference the input context like that, let's be safe and set the `.` explicitly